.gitlab-ci.yml

```diff
@@ -503,14 +503,13 @@ publish:production:s3:scripts:install-mender-sh:
     # Make copies in known destinations to be consumed by get.mender.io script:
     # * For "experimental" channel, we make a copy of the newest master package in the master directory
     # * For "stable" channel, we make a copy in a separate "latest" subdirectory of the latest tagged version
-
-    # Needs to be reworked, see MEN-5029
-    # - if is_build_tag ${MENDER_MONITOR_VERSION} || [ "${MENDER_MONITOR_VERSION}" == "master" ]; then
-    # -   aws s3 cp output/commercial/mender-monitor_*.deb s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_MONITOR}/master/mender-monitor_master-1_all.deb
-    # -   if test -f output/commercial/mender-monitor-demo*.deb; then
-    # -     aws s3 cp output/commercial/mender-monitor-demo*.deb s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_MONITOR}/master/mender-monitor-demo_master-1_all.deb
-    # -  fi
-    # - fi
+    - if [ "${MENDER_MONITOR_VERSION}" == "master" ]; then
+    -   aws s3 cp output/commercial/mender-monitor_*.deb s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_MONITOR}/master/mender-monitor_master-1+debian+buster_all.deb
+    -   if test -f output/commercial/mender-monitor-demo*.deb; then
+    -     aws s3 cp output/commercial/mender-monitor-demo*.deb s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_MONITOR}/master/mender-monitor-demo_master-1+debian+buster_all.deb
```

Comment:
same thing as below

Reply:
same thing as above :wink:

Reply:
hehe :) it is because I stand on my head so often.